src/libs/modulegroups.c

```diff
@@ -1109,18 +1109,16 @@ void init_presets(dt_lib_module_t *self)
   g_free(tx);
 
   // if needed, we add a new preset, based on last user config
-  if(!dt_conf_key_exists("plugins/darkroom/modulegroups_preset"))
-  {
```

Comment:
if you remove that check, that will means we redo the pre-3.4 preset retrieval at each startup, even for fresh install. and more problematic that will create the "previous_config"... presets for fresh install too.
As you make this retrieval-presets read-only in next lines, that should be enough...